PSI Food waste/Services/ProductService.cs

```diff
@@ -14,8 +14,9 @@ namespace PSI_Food_waste.Services
     public class ProductService
     {
         static List<Product> Products { get; set; }
+        static List<Product> Prs = new List<Product>();
```

Comment:
would be nice to have a more representing variable name, this one might confuse.